src/components/AppLayout/Header/components/Layout.tsx

```diff
@@ -95,9 +97,11 @@ const Layout = ({ classes, providerDetails, providerInfo }) => {
   return (
     <Row className={classes.summary}>
       <Col className={classes.logo} middle="xs" start="xs">
-        <Link to={WELCOME_ROUTE}>
-          <Img alt="Gnosis Safe" height={36} src={SafeLogo} testId="heading-gnosis-logo" id="safe-logo" />
-        </Link>
+        <Track {...SAFE_OVERVIEW_TRACKING_EVENTS.HOME}>
```

Comment:
Why do we need to track clicks on internal links?
The navigation should be tracked by page views/URL.

Reply:
That is correct. The information we gain with this is the button that was clicked and the page it button was clicked on. If that is not important I suggest we remove explicit tracking for internal links and solely rely on the pageview event. cc @johannesmoormann 
![Screenshot 2022-03-02 at 12 02 00](https://user-images.githubusercontent.com/5880855/156349516-c16ca1bc-aee0-42f5-96f0-9cf175b02ae2.png)

Reply:
We should only track internal links if we want to have the chain data or a payload alongside it.